packages/mobile/e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -20,6 +22,7 @@ export default HandleDeepLinkSend = () => {
     // 2. back button
     // there is a slight but important difference because with the back button
     // the activity gets destroyed and listeners go away which can cause subtle bugs
+    await device.reloadReactNative()
```

Comment:
do we still need this? or was this the fix that makes it less flaky?

Reply:
Currently this test is skipped. More investigation will be needed to determine on what is needed to make it run consistently without disrupting other test specs.